README.md

```diff
@@ -306,3 +309,4 @@ Copyright (c) 2017 [Ian Maurer], [Genomoncology LLC]
 [Example 02]: ./tests/ex02_compose_v3.2
 [Example 03]: ./tests/ex03_company
 [Example 04]: ./tests/ex04_contact
+[Example 05]: ./tests/ex06_json
```

Comment:
ex05 already exists, but it is not documented. I'm not sure whether you prefer not to expose that example/test or just forgot to do that. For now I used Example 05 as ex06